.github/workflows/edFi.database.admin Pull request.yml

```diff
@@ -0,0 +1,105 @@
+    - name: Checkout Ed-Fi-ODS-Implementation
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+          $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+          $is_pull_request_branch = 'False'
+          $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
+          if ($is_pull_request_branch -eq $true) {
+            git fetch origin ${{ env.CURRENT_BRANCH }}
+            git checkout ${{ env.CURRENT_BRANCH }}
+          }          
+    - name: Checkout Ed-Fi-ODS
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+          path: Ed-Fi-ODS/
+    - name: Is pull request branch exists in Ed-Fi-ODS
+      working-directory: ./Ed-Fi-ODS/
+      shell: pwsh
+      run: |
+          $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+          $is_pull_request_branch = 'False'
+          $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
+          if ($is_pull_request_branch -eq $true) {
+            git fetch origin ${{ env.CURRENT_BRANCH }}
+            git checkout ${{ env.CURRENT_BRANCH }}
+          }
+    - name: Start PostgreSQL
+      run: |
+        $pgService = Get-Service -Name postgresql*
+        Set-Service -InputObject $pgService -Status running -StartupType automatic
+        Start-Process -FilePath "$env:PGBIN\pg_isready" -Wait -PassThru
+    - name: Install sql-server-2019 & sqlpackage
+      working-directory: ./Ed-Fi-ODS-Implementation
+      shell: pwsh
+      run: |
+          choco install sql-server-2019  -y --params "'/IGNOREPENDINGREBOOT /IACCEPTSQLSERVERLICENSETERMS /Q /ACTION=install /INSTANCEID=MSSQLSERVER /INSTANCENAME=MSSQLSERVER /TCPENABLED=1 /UPDATEENABLED=FALSE /FEATURES=SQL,Tools'" --execution-timeout=$installTimeout
+          choco install sqlpackage
+    - name: Create Admin Database 
+      working-directory: ./Ed-Fi-ODS-Implementation
+      shell: pwsh
+      run: |
+          $ErrorActionPreference = 'Stop'
+          $PSVersionTable
+          . $env:GITHUB_WORKSPACE/Ed-Fi-ODS-Implementation/logistics/scripts/activities/build/create-database-package.ps1 -Output NugetPackages -DatabaseType Admin  -SQLPackage "C:\ProgramData\chocolatey\lib\sqlpackage\tools"
+    - name: Use NuGet
+      uses: nuget/setup-nuget@b2bc17b761a1d88cab755a776c7922eb26eefbfa # v1
+      with:
+        nuget-version: '5.x'
+    - name: Create NuGet package
+      working-directory: ./Ed-Fi-ODS-Implementation
```

Comment:
Similar to the step above, the working directory is not needed here, since you are specifying full paths to the package output and path to nuspec files, and then executing nuget, which gets installed globally.

Reply:
Removed it